src/edfi-paging-test/pyproject.toml

```diff
@@ -9,6 +9,13 @@ license = "Apache-2.0"
 python = "^3.9"
 ConfigArgParse = "^1.5.3"
 python-dotenv = "^0.19.2"
+requests-oauthlib = "^1.3.1"
+opnieuw = "^1.1.0"
+requests = "^2.27.1"
+types-requests = "^2.27.11"
+urllib3 = "^1.26.8"
+requests-mock = "^1.9.3"
+pytest-mock = "^3.7.0"
```

Comment:
Can `types-requests`, `requests-mock`, and `pytest-mock` move over to the dev dependencies?